NuGet/IronSoftware.Drawing.nuspec

```diff
@@ -58,16 +59,6 @@ For general support and technical inquiries, please email us at: support@ironsof
 				<dependency id="BitMiracle.LibTiff.NET" version="2.4.649" />
 				<dependency id="System.Memory" version="4.5.5" />
 			</group>
-			<group targetFramework="MonoAndroid10.0">
-				<dependency id="IronSoftware.Drawing.Abstractions" version="2025.2.6" />
-				<dependency id="SixLabors.ImageSharp" version="2.1.9" />
-				<dependency id="SixLabors.ImageSharp.Drawing" version="1.0.0" />
-				<dependency id="BitMiracle.LibTiff.NET" version="2.4.649" />
-				<dependency id="System.Memory" version="4.5.5" />
-				<dependency id="Microsoft.Maui.Graphics" version="7.0.92" />
-				<dependency id="SkiaSharp" version="2.88.7" />
-				<dependency id="SkiaSharp.Svg" version="1.60.0" />
-			</group>
```

Comment:
Don't forget to add `Microsoft.Maui.Graphics`, `SkiaSharp`, and `SkiaSharp.Svg` as dependencies for IronOcr for Android.